### PR TITLE
Userstory9

### PR DESCRIPTION
**Pull request by falkkreimeier**

### CODE REVIEW

File: src/components/MixedDrinks.spec.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    render(
+      <MixedDrinks
+        mixed={mixed}
+        name={name}
+        ingredience={ingredience}
+        preparation={preparation}
+      />
+    )
+
+    const text = screen.getByText('Shot - Wahnsinniger Bayleys')
```
Comment (StefanBehring) [inline src/components/MixedDrinks.spec.js:21]: you can do:
`const text = screen.getByText(`${mixed] - ${name}`)`
Same for **text2** and **text3**

That said, Lene does not like it when you use variables in the tests.

File: src/App.js [hunk 2]
```diff
@@ -59,38 +60,60 @@ function App({ data }) {
...
+                image={product.image}
+                logo={product.logo}
+                key={product.name}
+              />
+            ))}
+            <Shop onAddOrder={handleAddOrder} />
+          </Route>
+          <Route exact path="/motive">
+            {data.campaign.map(campaign => (
+              <CampaignCard image={campaign.image} />
```
Comment (macfrei) [inline src/App.js:83]: There is a `key` missing here.
```suggestion
              <CampaignCard image={campaign.image} key={campaign.image} />
```

File: src/App.js [hunk 3]
```diff
@@ -59,38 +60,60 @@ function App({ data }) {
...
+            ))}
+          </Route>
+          <Route>
+            {data.cocktails.map(cocktails => (
+              <MixedDrinks
+                name={cocktails.name}
+                mixed={cocktails.mixedDrinks}
+                ingredience={cocktails.ingredience}
+                preparation={cocktails.preparation}
+              />
```
Comment (macfrei) [inline src/App.js:93]: Same here, please add a `key`.

File: src/components/Footer.js [hunk 4]
```diff
@@ -10,33 +10,48 @@ function Footer() {
       <Link to="/motive">
         <Motiv src="./img/Logo.jpg" alt="Motive" />
       </Link>
+      <Link to="/mixedDrinks">
+        <Cocktails src="./img/Pinnchen.jpg" alt="Cocktails and Shots" />
```
Comment (macfrei) [inline src/components/Footer.js:14]: If you don't get the image source path from a data array, I would recommend you import this as it is recommended by React: 
```
import cocktailImg from "./img/Pinnchen.jpg"
...
   <Cocktails src={cocktailImg} alt="Cocktails and Shots" />
```

File: src/components/MixedDrinks.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components/macro'
+
+function MixedDrinks({ name, mixed, ingredience, preparation }) {
```
Comment (macfrei) [inline src/components/MixedDrinks.js:3]: Did you mean `ingredients`? 🙂

File: src/components/MixedDrinks.spec.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react'
+import MixedDrinks from './MixedDrinks'
+
+describe('MixedDrinks', () => {
+  const mixed = 'Shot'
+  const name = 'Wahnsinniger Bayleys'
+  const ingredience = 'Zutaten: Lakritzel und Bayleys'
```
Comment (macfrei) [inline src/components/MixedDrinks.spec.js:7]: I think this should be an array of ingredients.

**Pull request merged**